control_panel_api/views.py

```diff
@@ -270,12 +270,19 @@ class UserAppViewSet(viewsets.ModelViewSet):
     queryset = UserApp.objects.all()
     serializer_class = UserAppSerializer
 
+@api_view(['GET'])
+@permission_classes((ToolDeploymentPermissions,))
+@handle_external_exceptions
+def supported_tool_list(request):
+    tools = [{"name": n} for n in SUPPORTED_TOOL_NAMES]
+    return JsonResponse({"results": tools})
+
 
 @api_view(['POST'])
 @permission_classes((ToolDeploymentPermissions,))
 @handle_external_exceptions
 def tool_deployments_list(request, tool_name):
-    tool = Tool(tool_name)
+    tool = Tools[tool_name]()
```

Comment:
I wonder if `Tools` => `ToolsCatalog` or registry to be on the explicit side of things.